src/analyze.ts

```diff
@@ -110,16 +126,36 @@ export function parseStaticImport(matched: StaticImport): ParsedStaticImport {
   } as ParsedStaticImport;
 }
 
-export function findExports(code: string): ESMExport[] {
+export interface FindExportsOptions {
+  includeTypeExports?: boolean;
```

Comment:
Only for consistanct of options (when we support `findTypeImports`), maybe we can shorten the flag to something like `types: true` / `declarations: true` ?

Reply:
Maybe

```ts
findExports(code, {
  runtime: false,
  types: true
})
```

So it can be flexible to have type-only exports if needed?

Reply:
Hmm, this seems good. Then why not introduce `findTypeExports` returning `TSExport[]`? :D

Reply:
A lot of the normalisation is the same, and initially I didn't want too much duplicated code, but I can split it out into a shared function.